apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -1,277 +1,322 @@
+        };
+    }, [handleCoursesChange, handleCustomEventsChange, handleScheduleNamesChange, handleScheduleNoteChange]);
+
+    const scheduleUnits = useMemo(() => {
+        let result = 0;
+
+        for (const course of courses) {
             for (const section of course.sections) {
                 if (!isNaN(Number(section.units))) {
-                    totalUnits += Number(section.units);
+                    result += Number(section.units);
                 }
             }
         }
 
-        return totalUnits;
-    };
-
-    getGrid = () => {
-        const scheduleName = this.state.scheduleNames[AppStore.getCurrentScheduleIndex()];
-        const scheduleUnits = this.getTotalUnits();
-        const NOTE_MAX_LEN = 5000;
-
-        return (
-            <>
-                <div className={this.props.classes.titleRow}>
-                    <Typography variant="h6">{`${scheduleName} (${scheduleUnits} Units)`}</Typography>
-
-                    <div>
-                        <PopupState variant="popover">
-                            {(popupState) => (
-                                <>
-                                    <Button variant="outlined" color="inherit" {...bindTrigger(popupState)}>
-                                        Copy Schedule
-                                    </Button>
-                                    <Menu {...bindMenu(popupState)}>
-                                        {this.state.scheduleNames.map((name, index) => {
-                                            return (
-                                                <MenuItem
-                                                    key={index}
-                                                    disabled={AppStore.getCurrentScheduleIndex() === index}
-                                                    onClick={() => {
-                                                        copySchedule(index);
-                                                        popupState.close();
-                                                    }}
-                                                >
-                                                    Copy to {name}
-                                                </MenuItem>
-                                            );
-                                        })}
+        return result;
+    }, [courses]);
+
+    const scheduleName = useMemo(() => {
+        return scheduleNames[scheduleIndex];
+    }, [scheduleNames, scheduleIndex]);
+
+    return (
+        <Box>
+            <Box display="flex" width={1} justifyContent="space-between" marginY={2}>
+                <Typography variant="h6">{`${scheduleName} (${scheduleUnits} Units)`}</Typography>
+
+                <Box>
+                    <PopupState variant="popover">
+                        {(popupState) => (
+                            <>
+                                <Button variant="outlined" {...bindTrigger(popupState)}>
```

Comment:
```suggestion
                                <Button variant="outlined" color="inherit" {...bindTrigger(popupState)}>
```
Currently it's blue colored, but it should match what's on live